test/videouplinkbandwidthpolicy/NScaleVideoUplinkBandwidthPolicy.test.ts

```diff
@@ -488,6 +488,53 @@ describe('NScaleVideoUplinkBandwidthPolicy', () => {
         );
       }
     });
+
+    it('Scale to 360 if browser behavior returns disable 480p', () => {
+      class TestBrowserBehavior extends DefaultBrowserBehavior {
+        disable480pResolutionScaleDown(): boolean {
+          return true;
+        }
+      }
+      policy = new NScaleVideoUplinkBandwidthPolicy(
+        selfAttendeeId,
+        true,
+        undefined,
+        new TestBrowserBehavior()
+      );
+      policy.setIdealMaxBandwidthKbps(600);
+      policy.setTransceiverController(transceiverController);
+      transceiverController.setVideoInput(new MediaStreamTrack());
+      const numParticipants = 5;
+      const expectedParams = new DefaultVideoCaptureAndEncodeParameter(
+        320,
```

Comment:
How were these values calculated? I'm not to familiar with NScale, but 320x192 seems like an odd resolution.

Reply:
Oh this is just a dummy test case. We do not actually use it at all. The key factor is actual the scale down resolution.

Reply:
Ah that makes sense